docs/data/deployments.yml

```diff
@@ -221,3 +221,11 @@
   dask: dask-jobqueue
   jupyter: Single user notebook servers
   kind: hpc
+  
+- name: JASMIN Super-Data-Cluster
+  description: UK Science and Technology Facilities Council
+  link: http://www.jasmin.ac.uk/
+  platform: HPC
```

Comment:
Our currently supported Pangeo configuration actually uses Kubernetes on our OpenStack cloud. I wouldn't class it as HPC. That probably also affects the `kind` as well.

Reply:
Yeah I wasn't sure here. How about `OpenStack Cloud`?